### DISCUSSION TITLE
deploying the model after training

### DISCUSSION THREAD

**Discussion by sabiharustam** [mir-group/nequip#217] [Q&A] [answered]
I have trained the model successfully, however the deploying of the model is giving me some errors that I don't understand.  Here is the error message from the jupyter-notebook cell. I also get the same error if I try to use command line argument directly as well.
![image](https://user-images.githubusercontent.com/41701306/171729546-74ccab48-6746-4a12-bef4-1ec02948e998.png)

**Comment by Linux-cpp-lisp** [comment, marked as answer]
hi @sabiharustam ,

I think this missing a `--train-dir` after `build` and before the path to the training dir. That was a small change made in v0.5.4, see release changelog: https://github.com/mir-group/nequip/releases/tag/v0.5.4

**Reply by sabiharustam** [reply]
Thank you so much!!!! I was not aware of that change and got really confused.
